src/main/java/org/fcrepo/migration/PicocliMigrator.java

```diff
@@ -191,33 +194,46 @@ public Integer call() throws Exception {
             targetDir.mkdirs();
         }
 
-        // Fedora 6.0.0 expects a data directory at the top.
-        final File dataDir = targetDir.toPath().resolve("data").toFile();
-        if (!dataDir.exists()) {
-            dataDir.mkdirs();
+        if (workingDir == null) {
+            LOGGER.info("No working-dir option passed in - using current directory.");
+            workingDir = new File(System.getProperty("user.dir"));
+        }
+        if (!workingDir.exists()) {
+            workingDir.mkdirs();
         }
+        indexDir = new File(workingDir, "index");
 
-        // Create OCFL Storage dir
-        final File ocflStorageDir = new File(dataDir, "ocfl-root");
-        if (!ocflStorageDir.exists()) {
-            ocflStorageDir.mkdirs();
+        if (migrationType == MigrationType.FEDORA_OCFL) {
+            // Fedora 6.0.0 expects a data directory at the top.
+            final File dataDir = targetDir.toPath().resolve("data").toFile();
```

Comment:
Since `dataDir` isn't used for anything else now. This could be simplified by doing something like:

```java
ocflStorageDir = Files.createDirectories(targetDir.toPath().resolve("data").resolve("ocfl-root")).toFile();
```